randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -1726,6 +1726,13 @@ Extra - asset_id: collision_camera_025
   * Pickup 133; Category? Minor
   * Extra - actor_name: item_energyfragment
   * Extra - actor_def: actordef:actors/items/item_energyfragment/charclasses/item_energyfragment.bmsad
+  > Dock to EMMI Zone Exit East
+      All of the following:
+          # https://youtu.be/n8DTfVJoeFI
+          Ledge Warp (Intermediate)
+          Any of the following:
+              Lay Power Bomb
+              Damage Boost (Beginner) and Normal Damage ≥ 150
```

Comment:
Actually, i can see problems with this, where it could expect you to come down here and get something in one of the other two checks that require unmorphing, and then expect you to ledge warp back out, when you can't, I don't think this can actually be logical currently for that reason. Will need remote activation.